src/reconstruction/MatrixCoupling.jl

```diff
@@ -260,42 +260,58 @@ end
 
 # TODO see if some of these can be derived from AbstractModelWrapper
 @_change_bounds_fn MatrixCoupling Int inplace begin
-    change_bound!(model.lm, rxn_idx, lower = lower, upper = upper)
+    change_bound!(model.lm, rxn_idx, lower_bound = lower_bound, upper_bound = upper_bound)
 end
 
 @_change_bounds_fn MatrixCoupling Int inplace plural begin
-    change_bounds!(model.lm, rxn_idxs, lower = lower, upper = upper)
+    change_bounds!(model.lm, rxn_idxs, lower_bound = lower_bound, upper_bound = upper_bound)
 end
 
 @_change_bounds_fn MatrixCoupling String inplace begin
-    change_bound!(model.lm, rxn_id, lower = lower, upper = upper)
+    change_bound!(model.lm, rxn_id, lower_bound = lower_bound, upper_bound = upper_bound)
 end
 
 @_change_bounds_fn MatrixCoupling String inplace plural begin
-    change_bounds!(model.lm, rxn_ids, lower = lower, upper = upper)
+    change_bounds!(model.lm, rxn_ids, lower_bound = lower_bound, upper_bound = upper_bound)
```

Comment:
these can be done simpler using `;` in the right place:
```suggestion
    change_bounds!(model.lm, rxn_ids; lower_bound, upper_bound)
```

Reply:
(same in many places around in the file)

Reply:
will fix all these